social-seo-metatags.php

```diff
@@ -272,6 +264,17 @@ private function getFirstImage(): ?string
         }
       }
 
+      // Check if Antimatter theme header_image_file field is defined
+      if (isset($page->header()->header_image_file)) {
+        $tmp_images = array_replace([], $images);
+        foreach ($tmp_images as $image) {
+          if (basename($image->url()) != $page->header()->header_image_file) {
```

Comment:
Why this loop / implementation ? 
And why basename for one and not for header_image_file ?
Can you share the interface of `header_image_file` please ?

Reply:
Because I am not a PHP developer and I don't know a better approach. Sorry about this.